tests/test_journal.py

```diff
@@ -807,7 +807,7 @@ def test_submission(self, journal, openreview_client, test_client, helpers):
         ))
 
          # wait for process function delay (5 seconds) and check email has been sent
-        time.sleep(6)
+        helpers.await_queue(jobNames=set(['emailQueueStatus', 'gatherEmailsQueueStatus', 'jsEntityQueueStatus', 'pyEntityQueueStatus']), api_version=2)
```

Comment:
I think here we should wait until the process function associated with the edge is done, I can fix it